external-scripts.json

```diff
@@ -4,4 +4,5 @@
   "hubot-google",
   "hubot-redis-brain",
   "hubot-voting"
```

Comment:
Comma needed here as well.
It would help if you put a comma on the line below so that people editing this file in the future don't make the same mistake.